.eslintrc.js

```diff
@@ -17,6 +17,7 @@ const origamiServicesConfig = merge({},
 		"rules": {
 			"no-console": "off",
 			"no-await-in-loop": "off",
+			"no-warning-comments": "warn",
```

Comment:
This should be added to the [component eslint config](https://github.com/Financial-Times/origami-build-tools/blob/master/config/.eslintrc.js) instead of this eslint config. This eslint config is only for the OBT codebase.

Reply:
Makes sense. Closing.